template/src/navigation/StackNavigation.tsx

```diff
@@ -12,41 +13,33 @@ export const componentMatch = (stackName: string): Element | string => {
   }
 }
 
-const Stack = createNativeStackNavigator()
+const Stack = createNativeStackNavigator<AppStackParamList>()
 
-export const HomeNavigator = (): Element => {
+export const HomeNavigator = () => {
   return (
     <Stack.Navigator>
       <Stack.Screen
-        name={RouteKey.HomeScreen}
+        name={'HomeScreen'}
         component={screenMatch(RouteKey.HomeScreen)}
         options={optionsMatch(RouteKey.HomeScreen)}
       />
     </Stack.Navigator>
   )
 }
 
-export const AuthNavigator = (): Element => {
+export const AuthNavigator = () => {
   return (
     <Stack.Navigator>
       <Stack.Screen
-        name={RouteKey.LoginScreen}
+        name={'LoginScreen'}
         component={screenMatch(RouteKey.LoginScreen)}
         options={optionsMatch(RouteKey.LoginScreen)}
       />
       <Stack.Screen
-        name={RouteKey.SignUpScreen}
+        name={'SignUpScreen'}
```

Comment:
Create an Constant to store all screen name

Reply:
updated